Makefile

```diff
@@ -50,7 +50,7 @@ doc:
 	cd doc && make html O=-W
 
 docker-qa: | docker-qa-build
-	docker run --rm -v `pwd`:/gcovr $(QA_CONTAINER)
+	docker run --rm -e TEST_OPTS="$(TEST_OPTS)" -v `pwd`:/gcovr $(QA_CONTAINER)
```

Comment:
While this is certainly an improvement, I foresee shell quoting issues when the TEST_OPTS involve more complicated arguments, for example `make docker-qa TEST_OPTS='-k "xml and simple"'`. I think this can be avoided by using a target-specific export statement `docker-qa: export TEST_OPTS := $(TEST_OPTS)`, then:

```diff
-	docker run --rm -e TEST_OPTS="$(TEST_OPTS)" -v `pwd`:/gcovr $(QA_CONTAINER)
+	docker run --rm -e TEST_OPTS -v `pwd`:/gcovr $(QA_CONTAINER)
```

Reply:
I've overlooked the option to supress the value in the command line of docker to use the current host value. Updated PR.